### DISCUSSION TITLE
Query the number of parsers updates available

### DISCUSSION THREAD

**Discussion by rodhash** [nvim-treesitter/nvim-treesitter#5704] [General]
Couldn't find in the docs other Cmds than install / udpate/ uninstall .. the ideia is to build an indicator to inform when parsers updates are avail in order to know when to run the update Cmd.

**Comment by clason** [comment]
It's not supported, as it's not really useful: the only time a parser update _can be_ available is after updating the plugin -- in which case you can just run `:TSUpdate`, which will be a no-op if no update is available.
